manifest/armv7l/f/fontconfig.filelist

```diff
@@ -0,0 +1,86 @@
+/usr/local/include/fontconfig/fontconfig.h
+/usr/local/lib/libfontconfig.so
+/usr/local/lib/libfontconfig.so.1
+/usr/local/lib/libfontconfig.so.1.13.0
+/usr/local/lib/pkgconfig/fontconfig.pc
+/usr/local/share/fontconfig/conf.avail/05-reset-dirs-sample.conf
+/usr/local/share/fontconfig/conf.avail/09-autohint-if-no-hinting.conf
+/usr/local/share/fontconfig/conf.avail/10-autohint.conf
+/usr/local/share/fontconfig/conf.avail/10-hinting-full.conf
+/usr/local/share/fontconfig/conf.avail/10-hinting-medium.conf
+/usr/local/share/fontconfig/conf.avail/10-hinting-none.conf
+/usr/local/share/fontconfig/conf.avail/10-hinting-slight.conf
+/usr/local/share/fontconfig/conf.avail/10-no-antialias.conf
+/usr/local/share/fontconfig/conf.avail/10-scale-bitmap-fonts.conf
+/usr/local/share/fontconfig/conf.avail/10-sub-pixel-bgr.conf
+/usr/local/share/fontconfig/conf.avail/10-sub-pixel-none.conf
+/usr/local/share/fontconfig/conf.avail/10-sub-pixel-rgb.conf
+/usr/local/share/fontconfig/conf.avail/10-sub-pixel-vbgr.conf
+/usr/local/share/fontconfig/conf.avail/10-sub-pixel-vrgb.conf
+/usr/local/share/fontconfig/conf.avail/10-unhinted.conf
+/usr/local/share/fontconfig/conf.avail/10-yes-antialias.conf
+/usr/local/share/fontconfig/conf.avail/11-lcdfilter-default.conf
+/usr/local/share/fontconfig/conf.avail/11-lcdfilter-legacy.conf
+/usr/local/share/fontconfig/conf.avail/11-lcdfilter-light.conf
+/usr/local/share/fontconfig/conf.avail/20-unhint-small-vera.conf
+/usr/local/share/fontconfig/conf.avail/25-unhint-nonlatin.conf
+/usr/local/share/fontconfig/conf.avail/30-metric-aliases.conf
+/usr/local/share/fontconfig/conf.avail/35-lang-normalize.conf
+/usr/local/share/fontconfig/conf.avail/40-nonlatin.conf
+/usr/local/share/fontconfig/conf.avail/45-generic.conf
+/usr/local/share/fontconfig/conf.avail/45-latin.conf
+/usr/local/share/fontconfig/conf.avail/48-spacing.conf
+/usr/local/share/fontconfig/conf.avail/49-sansserif.conf
+/usr/local/share/fontconfig/conf.avail/50-user.conf
+/usr/local/share/fontconfig/conf.avail/51-local.conf
+/usr/local/share/fontconfig/conf.avail/60-generic.conf
+/usr/local/share/fontconfig/conf.avail/60-latin.conf
+/usr/local/share/fontconfig/conf.avail/65-fonts-persian.conf
+/usr/local/share/fontconfig/conf.avail/65-khmer.conf
+/usr/local/share/fontconfig/conf.avail/65-nonlatin.conf
+/usr/local/share/fontconfig/conf.avail/69-unifont.conf
+/usr/local/share/fontconfig/conf.avail/70-no-bitmaps.conf
+/usr/local/share/fontconfig/conf.avail/70-yes-bitmaps.conf
+/usr/local/share/fontconfig/conf.avail/80-delicious.conf
+/usr/local/share/fontconfig/conf.avail/90-synthetic.conf
+/usr/local/share/gettext/its/fontconfig.its
+/usr/local/share/gettext/its/fontconfig.loc
+/usr/local/share/locale/ka/LC_MESSAGES/fontconfig-conf.mo
+/usr/local/share/locale/ka/LC_MESSAGES/fontconfig.mo
+/usr/local/share/locale/zh_CN/LC_MESSAGES/fontconfig-conf.mo
+/usr/local/share/locale/zh_CN/LC_MESSAGES/fontconfig.mo
+/usr/local/share/xml/fontconfig/fonts.dtd
```

Comment:
We never had an armv7l fontconfig.filelist?

Reply:
no!